python/lsst/rucio/register/rucio_interface.py

```diff
@@ -296,7 +301,13 @@ def register_as_replicas(self, dataset_id, dataset_refs) -> None:
         """
         bundles = []
         for dataset_ref in dataset_refs:
-            bundles.append(self._make_bundle(dataset_id, dataset_ref))
+            if dataset_ref is list:
```

Comment:
```suggestion
            if type(dataset_ref) is list:
```
I think the complaint was about `==` with `type`, not to drop the `type` itself.